lib/alephant/preview/server.rb

```diff
@@ -41,6 +45,22 @@ class Server < Sinatra::Base
         render_component
       end
 
+      post "/components/batch" do
+        batch_components = []
+
+        ((JSON.parse(request.body.read, :symbolize_names => true) || {}).fetch(:components, [])).each do |component|
```

Comment:
@samfrench could we break this line out into multiple method calls please. 

e.g. something like...

```ruby
post "/components/batch" do
  ...

  batched_components.each do |component|
    # stuff
  end

  ...
end

def batched_components
  request_body.fetch(:components, [])
end

def request_body
  JSON.parse(request.body.read, :symbolize_names => true) || {}
end
```